plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -954,7 +954,7 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
         }
 
         if (bucketProperty.isPresent() && getAvroSchemaLiteral(tableMetadata.getProperties()) != null) {
-            throw new TrinoException(NOT_SUPPORTED, "Bucketing/Partitioning columns not spported when Avro schema literal is set");
+            throw new TrinoException(NOT_SUPPORTED, "Bucketing/Partitioning columns not supported when Avro schema literal is set");
```

Comment:
`Bucketing/Partitioning columns are not supported when Avro schema literal is set` ?

Reply:
yes, but also this existing class has a mixture of both `XXX not supported` and `XXX is/are not supported`. Feel welcome to introduce some consistency/logic in a fresh PR.